.goreleaser.yml

```diff
@@ -36,13 +36,9 @@ builds:
     ldflags:
       - -s -w -X github.com/flyteorg/flytestdlib/version.Version={{.Version}} -X github.com/flyteorg/flytestdlib/version.Build={{.ShortCommit}} -X github.com/flyteorg/flytestdlib/version.BuildTime={{.Date}}
 archives:
-  - name_template: '{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
-    replacements:
-      darwin: Darwin
-      linux: Linux
-      windows: Windows
-      386: i386
-      amd64: x86_64
+  - name_template: |-
+      {{ $replacements := dict "darwin" "Darwin" "linux" "Linux" "windows" "Windows" "386" "i386" "amd64" "x86_64" }}
+      {{ .ProjectName }}_{{ .Os }}_{{ or (index $replacements .Arch) .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}
```

Comment:
these replacements are weird in that some are for the `.Os` and others for the `.Arch`. So I'd prefer to have them separate (easier to see what's being replaced).

Reply:
Oh my bad. Missed that.